openhands/sdk/preset/default.py

```diff
@@ -1,40 +1,35 @@
 """Default preset configuration for OpenHands agents."""
 
-from openhands.sdk import AgentSpec, Tool, ToolSpec, create_mcp_tools
+from openhands.sdk import Agent
 from openhands.sdk.context.condenser import (
     LLMSummarizingCondenser,
 )
 from openhands.sdk.context.condenser.base import CondenserBase
 from openhands.sdk.llm.llm import LLM
-
-
-def get_default_tools(working_dir: str) -> list[Tool]:
-    """Get the default set of tools including MCP tools if configured."""
-    from openhands.tools import BashTool, FileEditorTool, TaskTrackerTool
-
-    tools = [
-        BashTool.create(working_dir=working_dir),
-        FileEditorTool.create(),
-        TaskTrackerTool.create(),
+from openhands.sdk.tool import ToolSpec, register_tool
+
+
+def get_default_tools(working_dir: str) -> list[ToolSpec]:
+    """Get the default set of tool specifications for the standard experience."""
+    from openhands.tools.browser_use import BrowserToolSet
+    from openhands.tools.execute_bash import BashTool
+    from openhands.tools.str_replace_editor import FileEditorTool
+    from openhands.tools.task_tracker import TaskTrackerTool
+
+    register_tool("BashTool", BashTool)
+    register_tool("FileEditorTool", FileEditorTool)
+    register_tool("TaskTrackerTool", TaskTrackerTool)
+    register_tool("BrowserToolSet", BrowserToolSet)
+
+    return [
+        ToolSpec(name="BashTool", params={"working_dir": working_dir}),
+        ToolSpec(name="FileEditorTool", params={}),
+        ToolSpec(
+            name="TaskTrackerTool", params={"save_dir": f"{working_dir}/.openhands"}
+        ),
+        ToolSpec(name="BrowserToolSet", params={}),
     ]
```

Comment:
```suggestion
def get_default_tools(working_dir: str, cli_mode: bool = False) -> list[ToolSpec]:
    """Get the default set of tool specifications for the standard experience."""
    from openhands.tools.execute_bash import BashTool
    from openhands.tools.str_replace_editor import FileEditorTool
    from openhands.tools.task_tracker import TaskTrackerTool

    register_tool("BashTool", BashTool)
    register_tool("FileEditorTool", FileEditorTool)
    register_tool("TaskTrackerTool", TaskTrackerTool)

    tool_specs = [
        ToolSpec(name="BashTool", params={"working_dir": working_dir}),
        ToolSpec(name="FileEditorTool", params={}),
        ToolSpec(
            name="TaskTrackerTool", params={"save_dir": f"{working_dir}/.openhands"}
        ),
    ]
    
    if not cli_mode:
        from openhands.tools.browser_use import BrowserToolSet
        register_tool("BrowserToolSet", BrowserToolSet)
        tool_specs.append(ToolSpec(name="BrowserToolSet", params={}))
    return tool_specs
```

Without this it seems browser use would still be included in the default agent spec that the CLI pulls

Reply:
actually i forgot to push this up earlier.. can you check the latest version?